lib/srv/sess.go

```diff
@@ -187,6 +197,20 @@ func (s *SessionRegistry) OpenSession(ch ssh.Channel, ctx *ServerContext) error
 
 		return nil
 	}
+
+	if ctx.srv.GetCreateHostUser() {
+		if ui := ctx.Identity.RoleSet.HostUsers(ctx.srv.GetInfo()); ui != nil {
+			userCloser, err := s.users.CreateUser(ctx.Identity.Login, ui)
+			if err != nil && !trace.IsAlreadyExists(err) {
+				log.Debug("Error creating user, already exists: ", err)
```

Comment:
This log entry is under `!trace.IsAlreadyExists(err)` condition so it probably shouldn't say "already exists"?